corehq/pillows/mappings/domain_mapping.py

```diff
@@ -211,6 +211,7 @@
                 'project_type': {'analyzer': 'comma', 'type': 'string'},
                 'published': {'type': 'boolean'},
                 'publisher': {'type': 'string'},
+                'allow_domain_requests': {'type': 'boolean'},
```

Comment:
I think neither minimal nor completeness is a priority for the mapping as there are other properties that aren't in the mapping. This should only affect what elastic search indexes, not what actually gets copied to the ES document.